src/main/kotlin/ink/abb/pogo/scraper/tasks/CatchOneNearbyPokemon.kt

```diff
@@ -110,6 +110,11 @@ class CatchOneNearbyPokemon : Task {
                 Log.red("Encounter failed with result: ${encounterResult.status}")
                 if (encounterResult.status == Status.POKEMON_INVENTORY_FULL) {
                     Log.red("Disabling catching of Pokemon")
+                    
+                    // Save previous value for restore
+                    ctx.pokemonInventoryFullStatus.first.set(settings.catchPokemon)
```

Comment:
Is this necessary as this is already set in https://github.com/jabbink/PokemonGoBot/pull/849/files#diff-4fe846c8e657c44271b65e94cae42312R52